### ISSUE TITLE
android.groovy determines root project path from user.dir property

### ISSUE THREAD

**Issue by ste7en** [jamonholmgren/react-native-colo-loco#15]
When running a gradle task from Fastlane, if the Fastfile is defined project-wide and not in the Android / iOS respective folders, gradle fails because the relative appPath I'm providing to `linkColocatedNativeFiles` is relative to the system property `user.dir` as in lines [17-18](https://github.com/jamonholmgren/react-native-colo-loco/blob/809cd910c1a11977efe30dfc2e6096ba62a1e93e/scripts/android.groovy#L17). 

Unfortunately neither the `rootDir` Gradle property can be used as it would be appended to `System.getProperty('user.dir')` path.

**Comment by jamonholmgren** [comment]
Sweep: What would you recommend we do with this?

(testing something)

**Comment by ste7en** [comment]
@jamonholmgren not really, I guess paths should be relative to the location of the script rather than `user.dir`.
I [asked](https://chat.openai.com/share/33a3031d-12db-4417-bd45-cbb53967de47) GPT 3.5 but didn't try it yet
